.gitignore

```diff
@@ -11,8 +11,6 @@ test/.project
 *.lock
 .idea/
 
-dist/
```

Comment:
Ok, happy with that. should we set it in the ignore list of the boilerplate too?